README.md

```diff
@@ -33,6 +33,8 @@ Use a ruby dependency manager like rbenv or asdf. The supported ruby version is
 
 `bin/rails s`
 
+**Note:** It's important to replace your valid GoCardless credentials in the .env file to fully leverage the application's functionalities.
```

Comment:
I would avoid recommending to overwrite the .env file as that's committed and a hypothetical developer could leak its gocardless credentials. According to `rails-dotenv` documentation the right approach would be to create an ignored file named: `env.local.development`

How about also linking this [table](https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use) in the README?

Reply:
Sure, will update it as per this.